src/ui/public/courier/data_source/build_query/__tests__/build_es_query.js

```diff
@@ -53,7 +53,7 @@ describe('build query', function () {
             { match_all: {} },
           ],
           filter: [
-            toElasticsearchQuery(fromKueryExpression('foo:bar'), indexPattern),
+            toElasticsearchQuery(fromKqlExpression('extension:jpg'), indexPattern),
```

Comment:
Are there three languages - Kuery, Kql and Lucene?  Or are KQL and Kuery the same thing? The PR description makes it seem like we are going to use KQL instead of Kuery but here the language on line 39 is kept as 'kuery' while the function name changed to 'fromKqlExpression'.